APIs/schemas/transporttype-response-schema.json

```diff
@@ -13,9 +13,15 @@
       ]
     },
     {
-      "description": "Any transport type URN base defined in the Transports register of the NMOS Parameter Registers",
+      "description": "Any transport type URN base defined by an AMWA specification in the Transports register of the NMOS Parameter Registers",
       "type": "string",
       "pattern": "^urn:x-nmos:transport:[a-zA-Z0-9_]+$"
+    },
+    {
+      "description": "Any manufacturer-specific transport type URN base defined in the Transports register of the NMOS Parameter Registers",
```

Comment:
Do we expect manufacturer namespaces to be registered in the transports register? I feel this whole area is thin in terms of interoperability. What would a controller be expected to do when it encounters a manufacturer namespace?

Reply:
I would assume that the controller then check if it knows about this transport and then use the vendor specific schemas about such transport. There may be an issue if multiple vendors claim the same namespace ...